### PR TITLE
[Feature] - Set remote SD URL with environment variable

### PR DESCRIPTION
**Pull request by sphuff**
**Motivation**:
Allows users to use a remote instance of the Automatic UI

**How to use**:
```
export SD_URL=http://104.143.3.146:10758 (example URL I was using)
// start app
uvicorn serverMain:app --reload
```
Or you can do the following:
* Add `set SD_URL=http://104.143.3.146:10758` in your `start_server.bat`

**How it works**:
All requests are proxied through the local python server. The remote SD server is then hit

### CODE REVIEW

File: server/python_server/serverMain.py [hunk 1]
```diff
@@ -9,9 +9,9 @@
 import time
 import serverHelper
 import prompt_shortcut
-def txt2ImgRequest(payload):
-    url = "http://127.0.0.1:7860"
 
+sd_url = os.environ.get('SD_URL', 'http://127.0.0.1:7860')
```
Comment (sphuff) [inline server/python_server/serverMain.py:13]: still defaults to the local instance on `7860`

**Comment by WadeWarren** [comment]
This is great – thanks!

**Comment by AbdullahAlfaraj** [comment]
this is awesome. 
all of the generation related request like (img2img, inpaint) are working
but the progressbar seem to be broken. it's stuck at 0%. 
I will keep testing it.

**Comment by sphuff** [comment]
@AbdullahAlfaraj ah I see what's happening - the server is only handling one request at a time, so the progress request isn't resolving until after the images are done. I should be able to do the image task in the background to fix - will update shortly

**Comment by sphuff** [comment]
Ok, done. I ending up adding `asyncio` and `httpx` to handle the async requests on the python side. So now I'm seeing it working with the progress request, as well as the txt2img/img2img

**Comment by AbdullahAlfaraj** [comment]
@sphuff fantastic work.

**Pull request merged**